torch/quantization/fake_quantize.py

```diff
@@ -65,9 +63,26 @@ def extra_repr(self):
             self.fake_quant_enabled, self.observer_enabled,
             self.scale, self.zero_point)
 
-default_fake_quant = FakeQuantize
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(FakeQuantize, self)._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'quant_min'] = self.quant_min
+        destination[prefix + 'quant_max'] = self.quant_max
+        destination[prefix + 'scale'] = self.scale
+        destination[prefix + 'zero_point'] = self.zero_point
+        destination[prefix + 'fake_quant_enabled'] = self.fake_quant_enabled
```

Comment:
is it the state we want to serialize in state_dict? state_dict logically is "learnable parameters", but not really the state of the model. For example we don't serialize `training` attribute today. Why do you need to serialize this runtime state?